mobile/src/libs/enums/navigation/main-screen-name.enum.ts

```diff
@@ -1,5 +1,5 @@
 const MainScreenName = {
-  HOME: 'Home',
+  DASHBOARD: 'Home',
```

Comment:
I think we should be naming our keys in the enum the same way we call our values

Reply:
I had task to rename Home to Dashboard, but leave only title as Home, so not to be confused later i renamed all , but left title